src/lib/BulkDataClient.ts

```diff
@@ -21,6 +21,7 @@ import { FileDownloadError }            from "./errors"
 import {
     assert,
     fhirInstant,
+    fileDownloadDelay,
```

Comment:
Looks like this import is no longer needed?

Reply:
Correct! Removed 👍